src/components/backend-ai-login.ts

```diff
@@ -1040,7 +1040,7 @@ export default class BackendAILogin extends BackendAIPage {
    * Show signup dialog. And notify message if API Endpoint is empty.
    * */
   private _showSignupDialog() {
-    this.api_endpoint = this.api_endpoint.trim();
+    this.api_endpoint = this.apiEndpointInput.value || this.api_endpoint.trim();
```

Comment:
@Sujin-Kim1  Please check the logic whether it works.

Reply:
If we don't set the `[apiEndpoint](https://github.com/lablup/backend.ai-webui/blob/main/configs/default.toml#L2)` option, it is not available to signup because of the empty endpoint value.
So I modified it not only to allow the user to set the specific endpoint with input value but also to allow the signup dialog to float even when the `apiEndpoint` option is not set.
However, if the policy allows signup only with the `apiEndpoint` option, I will revert.